src/style.css

```diff
@@ -1,3 +1,96 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
+
+  --header-height: 60px;
+  --font-size: 12px;
+  --link-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: auto;
+  max-width: 1200px;
+  min-width: 1024px;
+  width: 100%;
+  padding: 4.4px 50px;
+  box-sizing: border-box;
+  align-items: center;
+  outline: none;
+}
+
+.header:hover {
+  outline: 3px solid #00acdc;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  outline: none;
+}
+
+.logo:hover {
+  outline: 3px solid #00acdc;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav-list {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.list {
+  list-style-type: none;
+  margin: 0;
+}
+
+.list:not(.last-list) {
+  margin-right: 20px;
```

Comment:
The `.list:not(.last-list)` selector is not following the checklist recommendation to avoid using tag names for styling. Use class names that represent the content. Also, there seems to be a mismatch between the class names used in the HTML (`.nav__list`) and the ones here (`.nav-list`, `.list`). Ensure consistency in class naming.